rocrand/PKGBUILD

```diff
@@ -34,4 +34,6 @@ package() {
 /opt/rocm/rocrand/lib
 EOF
   install -Dm644 "$srcdir/rocRAND-rocm-$pkgver/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+  mkdir "$pkgdir/opt/rocm/include/hiprand"
```

Comment:
Let's use `install -d` here instead of `mkdir` so that file privileges are maintained.

Let's also add a comment that mentions some packages search for `# Some packages search for hiprand includes in /opt/rocm/include/hiprand`

Reply:
`-d` since we do not need to change permissions, right?

Reply:
The `-d` is for directory arguments. You can `man install` to find out more about `install`.

Reply:
I know, I read it :) just wanted to confirm that we do not need to change permissions